packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/data/ProjectPermissionsService.java

```diff
@@ -62,13 +62,11 @@ public List<Contributor> getContributors(final RebacProject rebacProject) throws
 	}
 
 	@CacheEvict(value = "projectcontributors", key = "#what.id")
-	public ResponseEntity<JsonNode> setProjectPermissions(
-			final RebacProject what, final RebacObject who, final String relationship) throws Exception {
+	public void setProjectPermissions(final RebacProject what, final RebacObject who, final String relationship)
+			throws Exception {
 		try {
 			what.setPermissionRelationships(who, relationship);
-			return ResponseEntity.ok().build();
-		} catch (final RelationshipAlreadyExistsException e) {
-			return ResponseEntity.status(HttpStatus.NOT_MODIFIED).build();
+		} catch (final RelationshipAlreadyExistsException ignore) {
```

Comment:
Why are we ignoring it?

Reply:
This exception is complaining that the relationship we are trying to set is already set.  Whilst this "should" not happen, it should not stop the process.